examples/src/main/java/com/mapbox/navigation/examples/core/ReplayActivity.kt

```diff
@@ -95,6 +102,25 @@ class ReplayActivity : AppCompatActivity(), OnMapReadyCallback {
             }
             true
         }
+        locationComponent = mapboxMap.locationComponent
+    }
+
+    @SuppressLint("RestrictedApi")
+    fun initLocationComponent(loadedMapStyle: Style, mapboxMap: MapboxMap) {
+        mapboxMap.moveCamera(CameraUpdateFactory.zoomTo(15.0))
+        mapboxMap.locationComponent.let { locationComponent ->
+            val locationComponentActivationOptions =
+                LocationComponentActivationOptions.builder(this, loadedMapStyle)
+                    // .useDefaultLocationEngine(true)
```

Comment:
NIT Remove commented out code?

Reply:
i'm working on ReplayActivity and am removing this. going to leave this PR as a pure revert so there's less conflicts

Reply:
ah ok removed it's fine lol